requirements.txt

```diff
@@ -0,0 +1,2 @@
+pre-commit
```

Comment:
If you do go with Conda, why install `pre-commit` and `black` from PyPI
rather than [`pre_commit`][pc] and [`black`][b] from Conda?

[pc]: https://anaconda.org/conda-forge/pre_commit
[b]: https://anaconda.org/conda-forge/black